CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Features:
 * VaultDynamicSecrets: CRD is extended with `Revoke` field which will result in the dynamic secret lease being revoked on rotation and CR deletion. Note: The VaultAuthMethod referenced by the VDS Secret must have a policy which provides `["update"]` on `sys/leases/revoke`. [GH-143](https://github.com/hashicorp/vault-secrets-operator/pull/143)
 * VaultAuth: Adds support for the JWT authentication method which either uses the JWT token from the provided secret reference, or a service account JWT token that VSO will generate using the provided service account. [GH-131](https://github.com/hashicorp/vault-secrets-operator/pull/131)
 
+Improvements:
+* VaultDynamicSecrets: Start lease renewal after 2/3 of lease duration [GH-170](https://github.com/hashicorp/vault-secrets-operator/pull/170)
```

Comment:
This change to renew at 2/3 may not be strictly necessary, especially with the fix for capped renewal below, but I left it in since it's similar to Vault Agent's behavior. Happy to remove it if y'all like.

Reply:
Yeah, like a `RenewalPercent` field that's out of 100? Probably restricted to 0-90 to leave room for up to 10% jitter. Trying that in 523b636.